spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/BaseCatalog.java

```diff
@@ -30,12 +30,14 @@
 import org.apache.spark.sql.connector.iceberg.catalog.ProcedureCatalog;
 import org.apache.spark.sql.util.CaseInsensitiveStringMap;
 
-abstract class BaseCatalog
+public abstract class BaseCatalog
```

Comment:
Wanted to use `BaseCatalog` instead of `TableCatalog` to support views in `BaseProcedure`.